### PR TITLE
Pytest support

### PR DESCRIPTION
**Pull request by fcanobrash**
Solves #28

### CODE REVIEW

File: scrapy_autounit/commands.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        print('No active Scrapy project')
+        sys.exit(1)
+
+    tests_dir = os.path.join(project_dir, 'autounit/tests')
+    for root, _, files in os.walk(tests_dir):
+        tests = [f for f in files if f.endswith('.py') and f != '__init__.py']
+        for test in tests:
+            test_path = os.path.join(root, test)
+            print('Updating test {}...'.format(test_path), end='')
+            with open(test_path, 'r+') as f:
```
Comment (andrewbaxter) [inline scrapy_autounit/commands.py]: Could this just call `write_test`?  I expect all the required info's in the bin file.

**Review by andrewbaxter — CHANGES_REQUESTED** [review]
Looks great!

File: scrapy_autounit/commands.py (hunk 1 above)
Comment (fcanobrash) [reply scrapy_autounit/commands.py]: Oh, right! I thought about it but wasn't sure where to get the `encoding` parameter from.
Totally forgot we were recording that with the response. Changing this, thanks!

Comment (fcanobrash) [reply scrapy_autounit/commands.py]: Wait, to get the `encoding` I need to unpickle, and for that I need the encoding too 😅
I'll take a further look tomorrow.

Comment (andrewbaxter) [reply scrapy_autounit/commands.py]: Ah, right :sweat_smile: good catch haha.

Yeah, that's tricky...

Still though I think scanning and modifying the .py file won't be realistic if there are more significant updates in the future...  Let me know if you don't have any ideas here and maybe we can brainstorm together.

**Pull request merged**